projects/angular/src/data/datagrid/render/main-renderer.ts

```diff
@@ -220,14 +226,18 @@ export class DatagridMainRenderer implements AfterContentInit, AfterViewInit, Af
   }
 
   private columnStateChanged(state) {
+    // eslint-disable-next-line eqeqeq
+    if (!this.headers || state.columnIndex == null) {
+      return;
+    }
```

Comment:
```suggestion
    if (!this.headers || state.columnIndex === null || state.columnIndex === undefined) {
      return;
    }
```

Reply:
using with two '==' checks for both null and undefined so it's okey.

Reply:
Yeah I know, my Idea was to remove the `// eslint-disable-next-line eqeqeq` exception.
It is better not to have any eslint disables exceptions unless is unavoidable and this is avoidable.

Also using `==` might affect some user application. 
Better be safe than sorry.